src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytQuantitativesErgebnisChoiceConverter.java

```diff
@@ -0,0 +1,31 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.virologischerbefund;
+
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytQuantitativesErgebnisChoice;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytQuantitativesErgebnisDvQuantity;
+import org.ehrbase.fhirbridge.ehr.opt.virologischerbefundcomposition.definition.ProAnalytQuantitativesErgebnisDvCount;
+import org.hl7.fhir.r4.model.Observation;
+
+public class ProAnalytQuantitativesErgebnisChoiceConverter {
+
+    public ProAnalytQuantitativesErgebnisChoice convertDvQuantity(Observation observation){
+
+        ProAnalytQuantitativesErgebnisDvQuantity proAnalytQuantitativesErgebnisDvQuantity = new ProAnalytQuantitativesErgebnisDvQuantity();
+
+        proAnalytQuantitativesErgebnisDvQuantity.setQuantitativesErgebnisMagnitude(observation.getValueQuantity().getValue().doubleValue());
```

Comment:
What happens if this is valuecodeable concept ? 
I cant see if this case is covered ?

Reply:
This was explained to me by the creator of the mapping:
In the case of ValueCodeableConcept only the Nachweistest is mapped after the code value. 
In the case of ValueQuantity Nachweistest is mapped to "Detected" and QuantitativesErgebnis will be mapped. QuantitativesErgebnis is an optional value.